examples/site/hello_world_content/hello_world_content.cc

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_hello_world_content]
+#include <google/cloud/functions/http_request.h>
+#include <google/cloud/functions/http_response.h>
+#include <nlohmann/json.hpp>
+#include <map>
+#include <string>
+
+namespace gcf = ::google::cloud::functions;
+
+std::map<std::string, std::string> parse_www_form_urlencoded(
+    std::string const& text);
+
+gcf::HttpResponse hello_world_content(gcf::HttpRequest request) {  // NOLINT
```

Comment:
Didn't you deal with the `NOLINT` issue in another PR?

Reply:
I started to, but it is not working yet.  I figured I could send this while I thought more about the other PR.